apps/aecontract/src/aect_state_tree.erl

```diff
@@ -226,21 +210,18 @@ verify_poi(Pubkey, Contract, Poi) ->
     end.
 
 verify_store_poi(Id, Store, IsLegalStoreFun, Poi) ->
-    %% Check separately absence of (invalid) empty key because Merkle
-    %% tree iterator-from-key, used in PoI lookup, does not return
-    %% initial key if present.
```

Comment:
I dropped this check. With the new representation empty keys are ignored, but I'm not completely sure what the check was for in the first place, so I don't know if dropping it has unintended consequences.